conjur/cli.py

```diff
@@ -168,6 +172,22 @@ def handle_list_logic(cls, list_data:ListData=None, client=None):
                                          list_data=list_data)
         list_controller.load()
 
+    @classmethod
+    def handle_hostfactory_logic(cls, args:list=None, client=None):
+        """
+            Method that wraps the hostfdactory call logic
+        """
+        if args.action_type == 'create_token':
+            hostfactory_logic = HostFactoryLogic(client)
+            create_token_data = CreateTokenData(hostfactoryid=args.hostfactoryid,
+                                                cidr=args.cidr,
```

Comment:
i would make attributes in CreateTokenData more readible maybe id, cidr, days, hours, minutes, count

Reply:
I think the attribute names are just very long and we should make them dev-friendly :)

Reply:
@sigalsax  I agree